models/users.js

```diff
@@ -119,16 +119,26 @@ const getSuggestedUsers = async (skill) => {
 
 /**
  * Fetches the data about our users
- * @param query { Object }: Filter for users data
+ * @param query { Object }: Filter for users
  * @return {Promise<userModel|Array>}
  */
 const fetchUsers = async (query) => {
+  const appendUsernamePrefixQuery = (dbQuery) => {
+    return dbQuery
+      .orderBy("username")
+      .startAt(query.search.toLowerCase().trim())
+      .endAt(query.search.toLowerCase().trim() + "\uf8ff");
```

Comment:
can you explain this endAt line?

Reply:
https://firebase.google.com/docs/database/rest/retrieve-data#range-queries

Reply:
thanks